src/test/java/nl.tudelft.graphalytics.graphlab/AlgorithmTest.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2015 Delft University of Technology
- *
+ * <p/>
```

Comment:
This will make `mvn license:check` fail. Please fix it and please run `mvn verify` before committing.